### PR TITLE
[13.0][MIG] sale_report_country_state: Migration to 13.0

### PR DESCRIPTION
**Pull request by joao-p-marques**
@Tecnativa
TT27872

@chienandalu @pedrobaeza can you review please?

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]

File: sale_report_country_state/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Copyright 2020 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Sale Report Filter by State",
+    "version": "13.0.1.0.0",
+    "author": "Tecnativa," "Odoo Community Association (OCA)",
```
Comment (victoralmau) [inline sale_report_country_state/__manifest__.py]:
```suggestion
    "author": "Tecnativa, Odoo Community Association (OCA)",
```

File: sale_report_country_state/i18n/es.po [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+#. module: sale_report_country_state
+#: model:ir.model,name:sale_report_country_state.model_report_all_channels_sales
+msgid "Sales by Channel (All in One)"
+msgstr ""
+
+#~ msgid "All sales orders grouped by sales channels"
+#~ msgstr "Todos los pedidos de venta agrupados por canal"
+
+#~ msgid "Sales Orders Statistics"
+#~ msgstr "Estadísticas de pedidos de ventas"
```
Comment (victoralmau) [inline sale_report_country_state/i18n/es.po]:
```suggestion
```

**Review by victoralmau — CHANGES_REQUESTED** [review]

File: sale_report_country_state/README.rst [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+Usage
+=====
+
+To filter by state you can either:
+
+#. Go to *Sales > Reporting > Sales*
+#. Group by *Partner's State* or filter for such criteria.
+
+#. Go to *Sales > Reporting > All Channels Sales Orders*
+#. Filter by State.
```
Comment (chienandalu) [inline sale_report_country_state/README.rst]: It doesn't exist anymore

File: sale_report_country_state/reports/report_all_channels_sales.py [hunk 4]
```diff
@@ -0,0 +1,33 @@
+# Copyright 2020 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class ReportAllChannelsSales(models.Model):
+    _inherit = "report.all.channels.sales"
+
+    state_id = fields.Many2one(
+        comodel_name="res.country.state", string="Partner's State", readonly=True,
```
Comment (chienandalu) [inline sale_report_country_state/reports/report_all_channels_sales.py]: Change to *Customer State* so it's grouped altogether with the other partner's properties

image

**Review by chienandalu — CHANGES_REQUESTED** [review]

**Comment by joao-p-marques** [comment]
Thanks @chienandalu @victoralmau 

Changes done!

**Review by victoralmau — APPROVED** [review]

**Review by chienandalu — APPROVED** [review]

**Pull request merged**